pyneuphonic/_clone_voices.py

```diff
@@ -0,0 +1,50 @@
+from typing import List
+
+import httpx
+
+from ._endpoint import Endpoint
+
+
+class CloneVoices(Endpoint):
+    def post(
```

Comment:
Put this entire function inside the `Voices` class. So it can be accessed via `client.voices.clone()` rather than `client.clone_voices.post()` as you have to access it now.